### PR TITLE
Change some words a bit

### PR DESCRIPTION
**Pull request by timneedham**
### Why:

I just read the "[Generating a user access token for a GitHub App](https://docs.github.com/en/apps/creating-github-apps/authenticating-with-a-github-app/generating-a-user-access-token-for-a-github-app)" page and spotted a few typos.

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Some grammar/typos.

### Check off the following:

- [ ] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by adhammahdi** [comment]
افتح حساب بنكي خاص بي وفواتير وأطلق اسحب

تم الإرسال من Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by cmwilson21** [comment]
@timneedham Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/apps/creating-github-apps/authenticating-with-a-github-app/authenticating-with-a-github-app-on-behalf-of-a-user.md [hunk 1]
```diff
@@ -24,10 +24,10 @@ Your app can make API requests on behalf of a user. API requests made by an app
 
 Similarly, if the request triggers a corresponding entry in the audit logs and security logs, the logs will list the user as the actor but will state that the "programmatic_access_type" is "GitHub App user-to-server token".
 
-To make an API request behalf of a user, the user must authorize your app. If an app is installed on an organization that includes multiple members, each member will need to authorize the app before the app can act on their behalf. An app does not need to be installed in order for a user to authorize the app.
+To make an API request on the behalf of a user, the user must authorize your app. If an app is installed on an organization that includes multiple members, each member will need to authorize the app before the app can act on their behalf. An app does not need to be installed in order for a user to authorize the app.
```
Comment (skedwards88) [inline content/apps/creating-github-apps/authenticating-with-a-github-app/authenticating-with-a-github-app-on-behalf-of-a-user.md:27]: To be consistent with the other places in this article:
```suggestion
To make an API request on behalf of a user, the user must authorize your app. If an app is installed on an organization that includes multiple members, each member will need to authorize the app before the app can act on their behalf. An app does not need to be installed in order for a user to authorize the app.
```

**Review by skedwards88 — COMMENTED** [review]
✨ Thank you for correcting these errors! I had one minor suggestion that I will incorporate, then we'll get this merged down for you.

**Review by skedwards88 — APPROVED** [review]

**Pull request merged**